docs/website/contents/for-developers/PreflightGuide.md

```diff
@@ -0,0 +1,181 @@
+    --topology /path/to/mainnet-topology.json \
+    --host-addr 0.0.0.0 --port 3001 \
+    --socket-path /path/to/node.socket
+```
+
+ - `--config`: Get the default Cardano mainnet configuration file [here][cardano mainnet conf].
+    - The logging system supports human-readable or JSON output (look for `ScText`/`ScJson`).
+      `ScJson` sometimes contains more information, and it can be handy for automatic processing via eg `jq`.
+    - Note that Genesis files are still mutable as long as their corresponding era is still in development (like Conway as of early 2024), so your configuration file might need changes once you update your node.
+ - `--topology`: This file defines how/when the node will connect to other nodes.
+   Get the default [here][cardano mainnet topo].
+
+   While syncing, it makes sense to set the valency, ie the number of nodes sampled from the given DNS name to connect to, to `1` for better performance.[^genesis-syncing]
+
+Make sure to store the log output somewhere.
+
+### Syncing to mainnet
+
+Now you are ready to start syncing to mainnet!
+
+During this long process, you can stare at the log output, and eg look for lines containing `Chain extended` or `AddedToCurrentChain`, which (at least) mention the slot number of the most recently added block.
+Bonus points for a plot of your sync progress over time!
+
+### Syncing faster 🚀
+
+As seen above, syncing is relatively slow, and you might be in a situation where you haven't caught up to mainnet in maybe a couple of months, but don't want to wait several hours.
+Here are two options:
+
+ - If do not mind a little extra clutter, it's possible to sync to mainnet in about two hours by using the [Mithril client][].
+   See [here][Mithril instructions] for instructions.
+ - Maybe someone you know has a very recent local copy of the mainnet ChainDB.
+   If you are sure you *really* trust them, ask them to send you their ChainDB (or at least the missing parts) via eg [Magic Wormhole][].
+
+## Understanding ledger snapshots and using db-analyser
+
+You need a local ChainDB for this step, see the previous step.
+
+[db-analyser][] is a tool maintained by the Consensus team to gather information and investigate problems (especially performance-related) by analyzing a local ChainDB, providing a variety of analysis passes.
+
+### Ledger snapshots
+
+A *ledger snapshot* is the CBOR serialization of the (extended[^extended-ledger-state]) *ledger state*, which represents the summary of all blocks up to a specific point[^glossary], the tip point or point of the last applied block, that is necessary to judge the validity of any block extending the last applied block.
+Here, "validity" refers most prominently to the ledger rules ("Does this person actually have enough money to make that payment?") of the [Cardano ledger].
+On disk, the file name of a ledger snapshot usually contains the slot number of its tip point.
+
+### Running db-analyser passes
```

Comment:
I remember that we note somewhere in the docs that you should probably use `db-analyser` with a copy of an immutable DB, because it might mess with the contents a bit

Reply:
Hmm, I locally always use db-analyser directly with the ImmDB of my local node, didn't have problems so far, but that doesn't mean there are no problems. Did you have a specific db-analyser functionality in mind that might be destructive?

Another point: Unless one uses some copy-on-write filesystem (I am still using boring ext4), using a copy of the ImmDB seems quite wasteful, given its current size of >160GB.